R/shell.R

```diff
@@ -26,7 +26,13 @@ start_shell <- function(master,
                         packages = NULL,
                         environment = NULL,
                         shell_args = NULL,
-                        app_jar = "sparkr-shell") {
+                        app_jar = NULL) {
+  # by default use sparkapi backend but allow us to pass "sparkr-shell" to test against sparkr
+  if (is.null(app_jar)) {
```

Comment:
Once that's done then I think we should remove the app_jar parameter
entirely

On Wed, Jul 20, 2016 at 7:44 PM, Javier Luraschi <notifications@github.com>
wrote:

> In R/shell.R
> <https://github.com/rstudio/sparkapi/pull/9#discussion_r71626740>:
>
> > @@ -26,7 +26,13 @@ start_shell <- function(master,
> >                          packages = NULL,
> >                          environment = NULL,
> >                          shell_args = NULL,
> > -                        app_jar = "sparkr-shell") {
> > +                        app_jar = NULL) {
> > +  # by default use sparkapi backend but allow us to pass "sparkr-shell" to test against sparkr
> > +  if (is.null(app_jar)) {
>
> Yes, I think that would make sense to expose as an option.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/sparkapi/pull/9/files/3d805129054f441782ac5727c5031bd3637d1880#r71626740>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXxw4uE5ICvwI36cQ2oUDXJWMWz--zks5qXrL2gaJpZM4JRT-1>
> .
>